adapters/axonix/axonix.go

```diff
@@ -4,25 +4,39 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"net/url"
+	"text/template"
 
 	"github.com/mxmCherry/openrtb/v15/openrtb2"
 	"github.com/prebid/prebid-server/adapters"
 	"github.com/prebid/prebid-server/config"
 	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
 	"github.com/prebid/prebid-server/openrtb_ext"
 )
 
 type adapter struct {
-	URI string
+	EndpointTemplate template.Template
 }
 
 func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters.Bidder, error) {
+	endpoint, err := template.New("endpointTemplate").Parse(config.Endpoint)
+	if err != nil {
+		return nil, fmt.Errorf("unable to parse endpoint url template: %v", err)
+	}
 	bidder := &adapter{
-		URI: config.Endpoint,
+		EndpointTemplate: *endpoint,
 	}
 	return bidder, nil
 }
 
+func (a *adapter) getEndPoint(ext *openrtb_ext.ExtImpAxonix) (string, error) {
```

Comment:
**Nitpick:** `P` in `getEndPoint` doesn't need to be capital. Please consider changing it to a lower-case `p`